SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialClassInfoManager.h

```diff
@@ -107,6 +107,7 @@ class SPATIALGDK_API USpatialClassInfoManager : public UObject
 	UClass* GetClassByComponentId(Worker_ComponentId ComponentId);
 	bool GetOffsetByComponentId(Worker_ComponentId ComponentId, uint32& OutOffset);
 	ESchemaComponentType GetCategoryByComponentId(Worker_ComponentId ComponentId);
+	const TArray<Schema_FieldId>& GetFieldIdsByComponentId(Worker_ComponentId ComponentId, const class FRepLayout& RepLayout);
```

Comment:
Nit - prefer to forward declare at the top of the file. Makes it clearer what you depend on.